client/reader/stream/x-post.jsx

```diff
@@ -149,8 +168,27 @@ class CrossPost extends PureComponent {
 						}
 					{ this.getDescription( post.author.first_name ) }
 				</div>
-			</Card> );
+				{ feedId && <QueryReaderFeed feedId={ +feedId } includeMeta={ false } /> }
```

Comment:
We _could_ do that, but elsewhere I think the query component makes the decision on whether to fetch the feed or not (`shouldFeedBeFetched`)?